types/src/on_chain_config/mod.rs

```diff
@@ -60,8 +73,9 @@ pub trait ConfigStorage {
 /// Trait to be implemented by a Rust struct representation of an on-chain config
 /// that is stored in storage as a deserialized byte array
 pub trait OnChainConfig: Send + Sync + DeserializeOwned {
+    const ADDRESS: &'static str = "0xA550C18";
```

Comment:
maybe comment on what this value is? Or ideally, can we define this value as a constant somewhere and use that?

Reply:
there's association_address() but it's not a const fn